Dockerfile.python

```diff
@@ -6,7 +6,7 @@ ENV GOBIN=/app/bin
 
 # Install the specific versions of the Go programs
 RUN go install github.com/akotlar/bystro-stats@1.0.0
-RUN go install github.com/bystrogenomics/bystro-vcf@2.1.1
+RUN go install github.com/bystrogenomics/bystro-vcf@2.1.2;
```

Comment:
Needed because 2.1.2 contains a fix allowing dosageOutput when no genotypes in the VCF file (empty dosageOutput results)